liwords-ui/src/gameroom/game_controls.tsx

```diff
@@ -85,14 +193,16 @@ type EGCProps = {
   onRematch: () => void;
   showRematch: boolean;
   onExamine: () => void;
+  onExportGCG: () => void;
 };
 
 const EndGameControls = (props: EGCProps) => {
   const [rematchDisabled, setRematchDisabled] = useState(false);
   return (
     <div className="game-controls">
       <Button>Options</Button>
-      <Button onClick={props.onExamine}>Export GCG</Button>
+      <Button onClick={props.onExamine}>Examine</Button>
+      <Button onClick={props.onExportGCG}>Export GCG</Button>
```

Comment:
we might want to make this part of Examine, so one of the options there. It's probably ok for right now.